tools/emterpretify.py

```diff
@@ -874,7 +874,7 @@ def process_code(func, code, absolute_targets):
         code[j+1] = global_vars[target]
       elif code[j] == 'absolute-value':
         # put the 32-bit absolute value of an abolute target here (correct except for adding eb to relocate at runtime)
-        absolute_value = absolute_start + absolute_targets[unicode(code[j+1])]
+        absolute_value = absolute_start + absolute_targets[str(code[j+1])]
```

Comment:
hmm, i doubt i wrote `unicode` here without a reason ;) but i don't remember what it was - isn't there some stuff in python 2 that doesn't work with `str`?

Reply:
I couldn't think of a 'must' reason, but probably this: `absolute_targets` is from json parser so its keys are typed as `unicode`. But this is not a 'must' because Python 2 still allows indexing by native string:

```py
import json
json.loads('{"mykey":true}')[b"mykey"] # returns True
```

And BTW `code[j+1]` is always a number.

Reply:
Oh right, it's a number, that's what confused me.